llvm/docs/RISCVUsage.rst

```diff
@@ -119,6 +119,7 @@ on support follow.
      ``Za128rs``       Supported (`See note <#riscv-profiles-extensions-note>`__)
      ``Za64rs``        Supported (`See note <#riscv-profiles-extensions-note>`__)
      ``Zacas``         Supported (`See note <#riscv-zacas-note>`__)
+     ``Zama16b``       Supported
```

Comment:
```suggestion
     ``Zama16b``       Supported (`See note <#riscv-profiles-extensions-note>`__)
```

Reply:
I guess you may also add Zama16b to `_riscv-profiles-extensions-note:`

Reply:
Am I supposed to add the extension name into the list in `riscv-profiles-extensions-note`?
Seems like not a extensions are listes there